### PR TITLE
Assertions and logging

### LINKED ISSUE #12: As a freshman/ anyone new to nus, I want to know if there is any direct bus to my target destination
...so that I do not need to change buses.

### PR DESCRIPTION
**Pull request by wamikamalik**
Final changes to route command
Some modification to help command
Fix #12

### CODE REVIEW

File: src/main/java/seedu/duke/logic/commands/RouteCommand.java [hunk 1]
```diff
@@ -22,13 +22,23 @@ public RouteCommand(String rawMessage) {
...
+        ArrayList<Bus> busOptions = BusData.possibleBuses(locations[0].trim(), locations[1].trim());
+        Ui.printRouteMessage(busOptions);
+    }
+
+    private void checkLocations(String[] locations) throws CustomException {
         if (locations[0].trim().length() == 0 || locations[1].trim().length() == 0) {
             throw new CustomException(ExceptionType.NO_LOCATIONS);
         } else if (locations[0].trim().equalsIgnoreCase(locations[1].trim())) {
+            assert locations[0].trim().length() != 0 : "Location 0 is empty.";
+            assert locations[1].trim().length() != 0 : "Location 1 is empty.";
```
Comment (EthanWong2212) [inline src/main/java/seedu/duke/logic/commands/RouteCommand.java]:
```suggestion
            assert !locations[0].isempty() : "Location 0 is empty.";
            assert !locations[1].isempty(): "Location 1 is empty.";
```

Comment (EthanWong2212) [reply src/main/java/seedu/duke/logic/commands/RouteCommand.java]: Looks good. Can consider using .isblank()

Comment (wamikamalik) [reply src/main/java/seedu/duke/logic/commands/RouteCommand.java]: Oh yes! That's a better way to do it.

**Review by EthanWong2212 — APPROVED** [review]
All looks good

**Pull request merged**